setup.cfg

```diff
@@ -34,8 +45,38 @@ all_files = 1
 [bdist_wheel]
 universal = 1
 
-[compile_catalog]
-directory = invenio_pages/translations/
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.7
+zip_safe = False
+install_requires =
+    invenio-base>=1.2.10
+    invenio-rest>=1.2.0
+
+[options.extras_require]
+tests =
+    pytest-cache>=1.0
+    invenio-app>=1.3.4,<2.0.0
+    pytest-invenio>=2.1.0,<3.0.0
+    invenio-db[postgresql,mysql]>=1.0.14,<2.0.0
+    sphinx>=4.5.0
+
+[options.entry_points]
+invenio_base.apps =
+    invenio_pages = invenio_pages:InvenioPages
+invenio_base.api_apps =
+    invenio_pages_rest = invenio_pages:InvenioPagesREST
+invenio_i18n.translations =
+    messages = invenio_pages
+invenio_db.models =
+    invenio_pages = invenio_pages.models
+invenio_base.blueprints =
+    invenio_pages = invenio_pages.views:blueprint
+invenio_base.api_blueprints =
+    invenio_pages_rest = invenio_pages.rest:blueprint
+invenio_db.alembic =
+    invenio_pages = invenio_pages:alembic
```

Comment:
missing?

```

[tool:pytest]
addopts = --black --isort --pydocstyle --doctest-glob="*.rst" --doctest-modules --cov=invenio_pages --cov-report=term-missing
testpaths = tests invenio_pages
```
I think we are missing `black` reformatting + ignore that commit

Reply:
Ah, right, will create a new PR for this :+1: